lib/process-services/src/lib/task-list/services/tasklist.service.ts

```diff
@@ -124,10 +124,10 @@ export class TaskListService {
      * Gets all tasks matching a query and state value.
      *
      * @param requestNode Query to search for tasks.
-     * @param state Task state. Can be "open" or "completed".
+     * @param state Task state. Can be "open" or "completed". State "all" applies to both "active" and "completed" tasks.
```

Comment:
"State "all" applies to both "**active**" and "completed" tasks." -  is that correct? Statement before this statement says: "Can be "**open**" or "completed"."